apps/site/pages/components/molecules/toast.mdx

```diff
@@ -0,0 +1,339 @@
+---
+description: Toasts are used to display notifications. They appear temporarily, and should contain a single line of text directly related to the operation performed.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/PaymentMethods___b0f344b254360daa53406cd631887adf.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { SectionItem, SectionList } from 'site/components/SectionItem'
+import { Button, Toast, useUI, UIProvider } from '@faststore/ui'
+import ToastUsage from 'site/components/Toast/ToastUsage'
+import { CircleWavyCheck, CircleWavyWarning } from '@faststore/components'
+
+<header>
+
+# Toast
+
+Toasts are used to display notifications. They appear temporarily, and should contain a single line of text directly related to the operation performed.
+
+</header>
+
+## Overview
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection direction="row">
+      <UIProvider>
+        <b>Simple Toast</b>
+        <ToastUsage message={'Simple Toast message.'} status={'INFO'} />
```

Comment:
Suggestion: remove the title (ex: "Simple Toast") and add it as button's label.

![image](https://user-images.githubusercontent.com/11613011/225990562-052e687f-bc21-4cfa-82a6-e0a9233bee8c.png)

Reply:
I think we can use the same approach as [SlideOver](https://faststore-site-git-feat-fs-632-toast-component-faststore.vercel.app/components/organisms/slide-over)

Reply:
Nice! I implemented @renatamottam approach [here](https://github.com/vtex/faststore/pull/1672/commits/a7090755322c5d0ecc43f474b25543e75d38aeca).